src/pages/Landing.tsx

```diff
@@ -0,0 +1,140 @@
+import { motion } from 'framer-motion';
+import RDSLogo from '../assets/rds-logo.svg';
+import {
+  FEATURES,
+  WHAT_IS_FEATURE_FLAG,
+  WHY_USE_FEATURE_FLAGS,
+} from '../constants/landing';
+import { ScrollIndicator } from '../components/landing/ScrollIndicator';
+
+const Landing = () => {
+  return (
+    <main className="min-h-screen bg-white" role="main">
+      <motion.div
+        className="relative z-0 flex min-h-[calc(100vh-64px)] flex-col items-center justify-center px-4 text-center"
+        initial={{ opacity: 0, y: 20 }}
+        animate={{ opacity: 1, y: 0 }}
+        transition={{ duration: 0.8 }}
+        role="banner"
+        data-testid="hero-section"
+      >
+        <div className="max-w-4xl">
+          <h1
+            className="mb-4 bg-gradient-to-r from-primary to-primary-light bg-clip-text pb-1 text-5xl font-bold text-transparent md:text-6xl"
+            data-testid="hero-title"
+          >
+            Welcome to Feature-Flag
+          </h1>
+          <div
+            className="flex items-center justify-center gap-3"
+            aria-label="Created by Real Dev Squad"
+          >
+            <span className="text-lg text-secondary-light">by</span>
+            <img src={RDSLogo} alt="RDS Logo" className="h-10 w-auto" />
```

Comment:
we should consider lazy-loading assets like the `RDSLogo` SVG to improve page performance

Reply:
We don't need lazy loading here - the logo is above the fold(seen on the screen without scroll) and SVGs are lightweight.

And since it's our local asset it will be bundled with your application during the build process. It becomes part of our JavaScript bundle rather than being a separate network request.